### DISCUSSION TITLE
Python tools installation failure

### DISCUSSION THREAD

**Discussion by dang-trinh156** [GEOS-DEV/GEOS#2779] [Q&A]
Dear GEOSX developers and users,
I tried to install python tools (https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/pythonTools/pythonAPI.html#python-tools-setup), and I successfully Built target geosx_python_tools, 
![image](https://github.com/GEOS-DEV/GEOS/assets/124559501/0bba32bd-083d-4736-9f9b-6ea10a8a6fe8)

However, when I load the libraries in both python and jupyter notebook, it appears to have this error
![image](https://github.com/GEOS-DEV/GEOS/assets/124559501/f637df6e-bbfe-4679-b45c-5fdc24fa256d)
and when i comment out the 2nd line (import pygeosx), the other libraries still have errors:
![image](https://github.com/GEOS-DEV/GEOS/assets/124559501/10278700-53a1-4d20-92b0-70197d297f68)

Can you please provide me where my installation fails? Given that I install it in a conda environment (called geosx), and I also modified Python3_EXECUTABLE  path to the directory in the CmakeCache.txt in build-anag-gcc-release (I chose ./host-configs/LBL/anag-gcc.cmake)
![image](https://github.com/GEOS-DEV/GEOS/assets/124559501/f390e8b6-3157-4d9f-9f07-44b91a92a1a8)

**Comment by tanha778** [comment]
Help me

**Reply by dang-trinh156** [reply]
@tanha778 you encouter this problem too? If yes, I haven't resolved it
